### PR TITLE
AA-476: Send org_key with tracking events

### PR DESCRIPTION
**Pull request by mikix**
Also convert the course tool click event from accidentally being a segment event into a log event, like it is on the LMS.

https://openedx.atlassian.net/wiki/spaces/DE/pages/2097119925/Improved+Event+Data+Fields+-+FY21Q2+Q3

### CODE REVIEW

File: src/courseware/social-share/SocialIcons.jsx [hunk 1]
```diff
@@ -48,7 +49,9 @@ function SocialIcons({
 
     const { administrator } = getAuthenticatedUser();
     sendTrackEvent(analyticsId, {
-      course_id: courseId,
+      org_key: org,
+      courserun_key: courseId,
+      course_id: courseId, // should only be courserun_key, but left as-is for historical reasons
```
Comment (ciduarte) [inline src/courseware/social-share/SocialIcons.jsx:54]: Will we be able to remove this in the future? Or will we be sending two courseId's forever

**Review by ciduarte — APPROVED** [review]
One small question, LGTM

File: src/courseware/social-share/SocialIcons.jsx (hunk 1 above)
Comment (mikix) [reply src/courseware/social-share/SocialIcons.jsx:54]: I don't know - that would be DE's call I guess, I don't know how they are merging data, or how long back they need/want to go?
I asked them if I should send both, and they were like, yes please. So :shrug:

Comment (doctoryes) [reply src/courseware/social-share/SocialIcons.jsx:54]: Having both fields in the event will help in making an asynchronous shift over to using the new field in various event-processing pipelines. The need for both fields won't be forever - but it's difficult to say how long they'll be needed.

**Pull request merged**